lib/game_manager.rb

```diff
@@ -20,6 +20,10 @@ def winningest_coach(season_id)
     @tracker.game_team_manager.winningest_coach(find_game_ids_of_season(season_id))
   end
 
+  def worst_coach(season_id)
+    @tracker.game_team_manager.worst_coach(find_game_ids_of_season(season_id))
+end
```

Comment:
Indentation error here for `end`. You can either commit here in this PR or catch it in your next one